go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/hashicorp/go-memdb v1.3.2
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/imdario/mergo v0.3.12
-	github.com/kong/go-kong v0.25.2-0.20220123181121-e965de22d701
+	github.com/kong/go-kong v0.27.0
```

Comment:
This and the other PR both have bumps.
Both are violating the principle of atomic change - which will also reduce the time for PR reviews.
Not requesting a change here but please avoid such changes.
I know that doing such changes increases the time it takes to do git management and plan, but if other smaller PRs are any proof, they overall benefit the code quality and everyone's personal productivity.